refactor/queuer/requirements.txt

```diff
@@ -0,0 +1,14 @@
+uvicorn==0.15.0
+python-dotenv==0.19.2
+pydantic==1.9.0
+fastapi==0.74.0
+nats-py==2.0.0
+python-multipart==0.0.5
+aiolimiter==1.0.0
+fastapi-socketio==0.0.9
+websockets==10.2
+requests==2.24.0
+mock==4.0.3
+pytest==7.1.1
```

Comment:
things like mock and pytest should remain in the developer requirements file

Reply:
Thanks. I have separated the developer requirements from distribution requirements in the `requirements` dir

Reply:
@wjcunningham7 @Emmanuel289 I can merge this if it's ready to go

Reply:
Yes please, it's ready to go @wjcunningham7 @scottwn.

Reply:
Had to update changelog and version to latest release.